src/ntfs/write.rs

```diff
@@ -32,6 +32,13 @@ use transit_model_collection::{Collection, CollectionWithId, Id, Idx};
 impl TryFrom<(&Ticket, &TicketPrice)> for PriceV1 {
     type Error = failure::Error;
     fn try_from((ticket, price): (&Ticket, &TicketPrice)) -> Result<Self> {
+        if price.currency != "Eur" {
```

Comment:
As discussed, it might be slightly more complex than that, we need to check if the input model is multi-currency or not. If only one currency is used, we can convert to FareV1. If multi-currencies, we need to panic.

Reply:
As discussed with @prhod and @papailio if the currency is not EUR, a warning is logged and the faulty price item is ignored